tests/components/abode/test_init.py

```diff
@@ -29,6 +30,38 @@ async def test_change_settings(hass):
         await hass.async_block_till_done()
         mock_set_setting.assert_called_once()
 
+    with patch(
+        "abodepy.Abode.set_setting", side_effect=AbodeException((13, "invalid setting"))
+    ):
+        await hass.services.async_call(
+            ABODE_DOMAIN,
+            SERVICE_SETTINGS,
+            {"setting": "confirm_snd", "value": "loud"},
+            blocking=True,
+        )
+
+        assert AbodeException
```

Comment:
What do we want to do here? If we want to assert that the service call raises this exception we should use `pytest.raises`.

Reply:
Sorry about that, I didn't mean to commit this test yet. I got carried away starting to work on more tests for another PR after this one to add some more tests. I'm going to remove this and `test_connection_error` as I don't want to grow this PR any further. I'll update/fix those tests with some more in a separate PR later.